### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mandar1jn**
added a space because it looked weird

Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](../CONTRIBUTING.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

**Comment by mandar1jn** [comment]
als added te discord link provided in the appsmith repository readme

**Comment by crweiner** [comment]
Hi @mandar1jn! Thank you for adding a link to their Discord server! I appreciate it! 😄

**Comment by mandar1jn** [comment]
Yes. Will do.

**Comment by mandar1jn** [comment]
@crweiner the link is already in the involvement section.

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
   - 1+ merged PR: Stickers
   - 3+ merged PR: T-Shirt
   - Top contributor Referral: WASD Keyboard.
-- **How to sign up**: Raise a PR and join us on discord.[Appsmith Hacktoberfest](https://github.com/appsmithorg/appsmith/discussions/863)
+- **How to sign up**: Raise a PR and join us on [discord](https://discord.gg/rBTTVJp). [Appsmith Hacktoberfest](https://github.com/appsmithorg/appsmith/discussions/863)
```
Comment (crweiner) [inline README.md:44]: If you wish to make this change, please also do it below in the Least Involvement to Most Involvement section, such as line number 373 below.

**Comment by crweiner** [comment]
Closing due to inactivity. @mandar1jn, if you see any typos that need correcting, please feel free to submit a new PR. 👍

**Pull request closed without merging**